tests/test_oauth2_reddit.py

```diff
@@ -108,6 +110,24 @@ def test_scope_modlog(self):
         self.assertEqual(num, len(list(result)))
 
     @betamax()
```

Comment:
Do you think we should continue writing tests for the specific scopes like this? I started it down this path to provide really simple tests for each scope. However, as we remove the login functionality, it would be great to have tests in place for each of the methods with both login and their oauth counter parts.